apps/wallet/src/features/Send/TransactionResult/index.tsx

```diff
@@ -22,6 +23,22 @@ const TransactionResult: FC<Props> = ({ navigator }) => {
 		navigator.slideTo(0);
 	};
 
+	const handleBackToHome = () => {
+		navigationRef.reset({
+			index: 1,
+			routes: [
+				{ name: 'Dashboard' },
+				{
+					name: 'Dashboard',
+					params: {
+						screen: 'Home',
+					},
+				},
+			],
+		});
```

Comment:
this should be handled/implement inside utils/navigation, then other places could re-use the same method.

Reply:
I think the `reset` method of react native navigation is generic enough to use instead of rewriting it in `utils/navigation`